source/plot/LinePlotData2D.ooc

```diff
@@ -9,33 +9,37 @@ LineStyle: enum {
 	Dashed
 	Dotted
 }
-
 LinePlotData2D: class extends PlotData2D {
 	lineStyle: LineStyle { get set }
-
+	dataSeries: VectorList<FloatPoint2D> { get set }
 	init: func ~default (lineStyle := LineStyle Solid) {
-		super()
+		super("", ColorBgra new())
+		this dataSeries = VectorList<FloatPoint2D> new()
 		this lineStyle = lineStyle
 	}
-
-	init: func ~dataSeries (dataSeries: VectorList<FloatPoint2D>, label := "", colorBgra := ColorBgra new(), lineStyle := LineStyle Solid) {
-		super(dataSeries, label, colorBgra)
+	init: func ~dataSeries (=dataSeries, label := "", colorBgra := ColorBgra new(), lineStyle := LineStyle Solid) {
+		super(label, colorBgra)
 		this lineStyle = lineStyle
 	}
-
-	init: func ~color (dataSeries: VectorList<FloatPoint2D>, colorBgra: ColorBgra, lineStyle := LineStyle Solid) {
-		this init(dataSeries, "", colorBgra, lineStyle)
+	init: func ~color (=dataSeries, colorBgra: ColorBgra, lineStyle := LineStyle Solid) {
+		super("", colorBgra)
+		this lineStyle = lineStyle
 	}
-
 	init: func ~twoFloatSeries (xSeries, ySeries: VectorList<Float>, label := "", colorBgra := ColorBgra new(), lineStyle := LineStyle Solid) {
```

Comment:
Yes, it should be consistent, so `xSeries` and `ySeries` should be freed here, I think.